### PR TITLE
[monorepo]: update dependencies and linters and fix branch

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

File: linters/python/.pylintrc
Comment (AnthonyLaw) [inline linters/python/.pylintrc:1]: Should we use a submodule?
something like this https://github.com/symbol/product/blob/dev/init.sh

**Review by AnthonyLaw — APPROVED** [review]
👍🏼

**Review by Wayonb — APPROVED** [review]
👍🏾

File: linters/python/.pylintrc
Comment (Jaguar0625) [reply linters/python/.pylintrc:1]: probably, but up to @Wayonb when he integrates jenkins

**Pull request merged**